utils/scripts/get_pr_merged_labels.sh

```diff
@@ -34,7 +34,7 @@ if [[ $CI_COMMIT_MESSAGE =~ ($PR_PATTERN) ]]; then
         echo "The PR $PR_NUMBER doesn't contain the 'build-buddies-images' label "
     else
         echo "The PR $PR_NUMBER contains the 'build-buddies-images' label. Launching the images generation process "
-        ./utils/build/build_tracer_buddies.sh --push
+        ./utils/build/build_tracer_buddies.sh --push --platform linux/amd64
```

Comment:
Why this option ?

Reply:
tl;dr Explicit is better than implicit.


Previously it was inferred from the runner architecture:

> The default value is the platform of the BuildKit daemon where the build runs.

https://docs.docker.com/reference/cli/docker/buildx/build/#platform

By using the golden path runners we shift the build to remote workers, so using the `--platform` arg make it clear which architecture we're building for and what pool of workers we're targeting.